itest/lnd_multi-hop_test.go

```diff
@@ -2068,12 +2078,6 @@ func runMultiHopHtlcAggregation(ht *lntest.HarnessTest,
 	}
 
 	switch c {
-	// In case this is a non-anchor channel type, we must mine 2 blocks, as
-	// the nursery waits an extra block before sweeping. Before the blocks
-	// are mined, we should expect to see Bob's commit sweep in the mempool.
-	case lnrpc.CommitmentType_LEGACY:
```

Comment:
Same as above, this was never tested because we only test channels with anchor outputs.